packages/common/client-utils/src/test/mocha/events.spec.ts

```diff
@@ -358,29 +355,28 @@ export class MyExposingClass {
 }
 
 /**
- * Validates that an error thrown by assert() function has the expected message.
+ * Validates that an error has the expected message.
  *
- * @param error - The error object thrown by `assert()` function.
  * @param expectedErrorMessage - The message that the error object should match.
- * @returns `true` if the message in the error object that was passed in matches the expected
- * message. Otherwise it throws an error.
+ * @returns an error validation function suitable for use with NodeJS's `assert.throws()`.
  *
  * @remarks
- * Similar to {@link @fluidframework/test-runtime-utils#validateAssertionError}.
- *
- * @internal
+ * This does not work for asserts which get "tagging" see {@link @fluidframework/test-runtime-utils#validateAssertionError} for that.
+ * This avoids a dependency on `@fluidframework/test-runtime-utils` since this package cannot use it.
  */
-function validateAssertionError(error: Error, expectedErrorMsg: string | RegExp): boolean {
-	const actualMsg = error.message;
-	if (
-		typeof expectedErrorMsg === "string"
-			? actualMsg !== expectedErrorMsg
-			: !expectedErrorMsg.test(actualMsg)
-	) {
-		// This throws an Error instead of an AssertionError because AssertionError would require a dependency on the
-		// node assert library, which we don't want to do for this library because it's used in the browser.
-		const message = `Unexpected assertion thrown\nActual: ${error.message}\nExpected: ${expectedErrorMsg}`;
-		throw new Error(message);
-	}
-	return true;
+function validateError(expectedErrorMsg: string | RegExp): (error: Error) => true {
```

Comment:
This copy of validateAssertionError has no assert specific logic in it (other than the I think incorrect comment about AssertionError below), so I clarified its name and docs a bit while making it use the new style.